### PR TITLE
Fix update admin

### PR DESCRIPTION
**Pull request by jerry-yu**
Fix update admin

### CODE REVIEW

File: scripts/contracts/src/permission_management/Authorization.sol [hunk 1]
```diff
@@ -34,16 +34,36 @@ contract Authorization is IAuthorization, ReservedAddrPublic {
...
+        address oldAdmin = allAccounts[0];
+        if (_newAdmin == oldAdmin) {
+           return;
+        }
+        allAccounts[0] = _newAdmin;
+        allAccounts.push(oldAdmin);
+        for (uint8 i;i < builtInPermissions.length; i++) {
+            _setAuth(_newAdmin, builtInPermissions[i]);
+            // keep send tx and create contract
+            if (i != 13 && i!= 14)
```
Comment (leeyr338) [inline scripts/contracts/src/permission_management/Authorization.sol]: what meaning for `13` and `14` ? 
It is better to use a constant define?

Comment (jerry-yu) [reply scripts/contracts/src/permission_management/Authorization.sol]: It is not c++

File: scripts/contracts/src/permission_management/Authorization.sol [hunk 2]
```diff
@@ -188,4 +208,15 @@ contract Authorization is IAuthorization, ReservedAddrPublic {
...
+    /// @notice Private: delete the permission to the account
+    function _cancelAuth(address _account, address _permission)
+        private
+        returns (bool)
+    {
+        AddressArray.remove(_account, accounts[_permission]);
+        AddressArray.remove(_permission, permissions[_account]);
+        emit AuthCanceled(_account, _permission);
+        return true;
+    }
```
Comment (ouwenkg) [inline scripts/contracts/src/permission_management/Authorization.sol]: https://github.com/cryptape/cita/blob/develop/scripts/contracts/src/permission_management/Authorization.sol#L63, Why not use `_cancelAuth()` directly ?

Comment (jerry-yu) [reply scripts/contracts/src/permission_management/Authorization.sol]: has no right

**Pull request merged**